lib/core/constants/asset_images.dart

```diff
@@ -1,3 +1,3 @@
 class AssetImages {
-  static const footballSplash = 'assets/football_splash4.png';
+  static const String footballSplash = 'assets/football_splash4.png';
```

Comment:
not an error, but the type annotation is not needed here. The compiler will know.